website/public/team.html

```diff
@@ -1,95 +1,91 @@
+      </div>
+    </nav>
 
-	<li><a href="team.html">Team</a></li>
-      </ul>
-    </div>
-    <div class="search pull-right">
-      <form action="https://sbpy.readthedocs.io/en/latest/search.html" method="get">
-	<input type="text" name="q" placeholder="Search Documentation" />
-	<input type="hidden" name="check_keywords" value="yes" />
-	<input type="hidden" name="area" value="default" />
-      </form>
-    </div>
-  </nav>
-  
-  <section class="whatsnext">
+    <section class="whatsnext">
       <h1>The sbpy Team</h1>
       <p>The development of <em>sbpy</em> is a team effort that would
-	not be possible without the help of supporters that provide,
-	test, and document code:</p>
+        not be possible without the help of supporters that provide,
+        test, and document code:</p>
 
-	<p align="center"><a href="https://michaelmommert.wordpress.com">
-	    Michael Mommert</a>, Lowell Observatory</p>
-	<p align="center"><a href="https://www.astro.umd.edu/~msk/">
-	    Michael S. P. Kelley</a>, University of Maryland</p>
-	<p align="center">Miguel de Val-Borro, Planetary Science Institute</p>
-	<p align="center"><a href="https://jyli.me/">Jian-Yang Li</a>, Planetary Science Institute</p>
-	<p align="center"><a href="https://www.linkedin.com/in/giannina-guzman-3965bb166/">Giannina Guzman</a>, Villanova University/Lowell Observatory</p>
-	<p align="center">Brigitta Sipőcz, University of Washington</p>
-	<p align="center">Josef Ďurech, Charles University Prague</p>
-	<p align="center">Mikael Granvik, University of Helsinki</p>
-	<p align="center">Will Grundy, Lowell Observatory</p>
-	<p align="center">Nick Moskovitz, Lowell Observatory</p>
-	<p align="center">Antti Penttilä, University of Helsinki</p>
-	<p align="center">Nalin Samarasinha, Planetary Science Institute</p>
+      <p align="center"><a href="https://www.astro.umd.edu/~msk/">
+          Michael S. P. Kelley</a>, University of Maryland (Co-Lead Developer)</p>
+      <p align="center"><a href="https://jyli.me/">Jian-Yang Li</a>, Planetary Science Institute (Co-Lead
+        Developer)</p>
+      <p align="center"><a href="https://mommermi.github.io/">
+          Michael Mommert</a>, University of St. Gallen, Switzerland (Initial Project Leader)</p>
+      <p align="center">Miguel de Val-Borro, Planetary Science Institute</p>
+      <p align="center"><a href="https://www.linkedin.com/in/giannina-guzman-3965bb166/">Giannina Guzman</a>,
+        Villanova University/Lowell Observatory</p>
+      <p align="center">Brigitta Sipőcz, University of Washington</p>
+      <p align="center">Josef Ďurech, Charles University Prague</p>
+      <p align="center">Mikael Granvik, University of Helsinki</p>
+      <p align="center">Will Grundy, Lowell Observatory</p>
+      <p align="center">Nick Moskovitz, Lowell Observatory</p>
+      <p align="center">Antti Penttilä, University of Helsinki</p>
+      <p align="center">Nalin Samarasinha, Planetary Science Institute</p>
       <p>&nbsp; </p>
```

Comment:
I can't remember clearly, but has Henry Hsieh contributed?

Also I found many on the list have websites.  Should we link all we can find?

Reply:
He's not in the contributors list https://github.com/NASA-Planetary-Science/sbpy/graphs/contributors  nor on the proposal.

I've added links for everyone.